profile_dists/utils.py

```diff
@@ -207,9 +212,9 @@ def combine_header(h1,h2):
     return combined
 
 def create_col_map(columns):
-    column_mapping ={}
+    column_mapping = {}
     for col in columns:
-        column_mapping[col] = {"0":0}
+        column_mapping[col] = {}
```

Comment:
These seemed unused (and didn't show up at all when called from arborator). I noticed that 0 is the default flag for missing alleles, but it's still passing tests that handle empty/missing values (`data_empty_row_*.tsv`).